File: app/src/main/java/com/yzyfdf/libsample/api/HostType.java
package com.yzyfdf.libsample.api;


/**
 * 域名管理
 */
public enum HostType {

    Industry(ApiConstant.industrry, 1),
    UserCenter(ApiConstant.userCenter, 2);

    public String mHost;
    public int mIndex;

    HostType(String host, int index) {
        mHost = host;
        mIndex = index;
    }
}

File: app/src/main/java/com/yzyfdf/libsample/api/HostType.kt
package com.yzyfdf.libsample.api


/**
 * 域名管理
 */
enum class HostType private constructor(var mHost: String, var mIndex: Int) {

    Industry(ApiConstant.industrry, 1),
    UserCenter(ApiConstant.userCenter, 2)
}
